packages/docs/src/components/app/hljs-darcula.scss

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
why is this needed?

Reply:
For the dark code theme, white code in the dark theme looks ugly